packages/.gitignore

```diff
@@ -0,0 +1 @@
+**/*.patch.bz2
```

Comment:
```suggestion
*.patch.bz2
```
I think this simpler version will work without having to include the wildcard for directories.